custom/icds_reports/ucr/reports/mobile_mpr_2a_deaths.json

```diff
@@ -0,0 +1,385 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "icds-test",
+    "icds-sql",
+    "icds-cas",
+    "icds-cas-sandbox"
```

Comment:
should just need `icds-dashboard-qa` and `icds-cas` unless you guys are using other domains for testing. (good to keep these small as it reduces number of these kept in memory) I think all the others are test domains from long ago

Reply:
Should these also be removed from all the other reports?

Reply:
(I'll stick to the ones I'm editing either way, just to avoid scope creep)

Reply:
Yeah the other ones can be removed from old reports. I don't think anyone uses those domains anymore (and if htey did some reports would probalby be broken)